p2p/discover/v5wire/msg.go

```diff
@@ -59,7 +61,7 @@ type (
 		Nonce Nonce
 	}
 
-	// Whoareyou contains the handshake challenge.
+	// WHOAREYOU contains the handshake challenge.
```

Comment:
Why change these? They are "correct" as they were. You could change to 

// Whoareyou contains the handshake challenge, representing the WHOAREYOU message

Reply:
I made these upper case because packet names are always uppercase (in the spec, and also in logs).